riscv/runtime/src/print.rs

```diff
@@ -0,0 +1,79 @@
+use core::arch::asm;
+use core::fmt;
+
+// #[macro_export]
+// macro_rules! print {
+//     ($($arg:tt)+) => (print_args(format_args!( $($arg)+)))
+// }
+
+// TODO turn this into a macro
+pub fn print(args: fmt::Arguments) {
+    let mut buf = [0u8; 1024];
```

Comment:
Let me actually change this: We can use "potentially uninitialized" - then it should be much faster.